test/testdata/workspaces/hello-nexus-app/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM busybox
+
+ARG nexusUrl
+ARG nexusUsername
+ARG nexusPassword
+ARG nexusAuth
+ARG nexusUrlWithAuth
+ARG nexusHost
+
+
+RUN echo "nexusUrl=${nexusUrl}" >> message.txt
+RUN echo "nexusUsername=${nexusUsername}" >> message.txt
+RUN echo "nexusPassword=${nexusPassword}" >> message.txt
+RUN echo "nexusAuth=${nexusAuth}" >> message.txt
+RUN echo "nexusUrlWithAuth=${nexusUrlWithAuth}" >> message.txt
+RUN echo "nexusHost=${nexusHost}" >> message.txt
```

Comment:
@henninggross Micro optimization (probably not important): We could use a heredoc here, e.g. 

```sh
cat > message.txt << EOF
nexusUrl=${nexusUrl}
nexusUsername=${nexusUsername}
nexusPassword=${nexusPassword}
nexusAuth=${nexusAuth}
nexusUrlWithAuth=${nexusUrlWithAuth}
nexusHost=${nexusHost}
```

thus condense the six RUN statements into one. This would save us five layers.

Reply:
I wasn't sure whether or how exactly a heredoc works in a Dockerfile.

Reply:
@henrjk Just checked, to be able to use heredocs, you actually have to include a magic comment in the `Dockerfile` 😐  :

```Dockerfile
# syntax=docker/dockerfile:1.3-labs
FROM busybox
...
```

Reply:
@kuebler it looks like one cannot use variables in the heredoc, but perhaps it is a pilot error. 
Feel free to contribute that change if you know how to get the syntax right as it might be a good showcase for other scenarios where it could make the Dockerfile easier to author.